testing/jsonnet/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Benjamin Staffin <benley@gmail.com>
+# Maintainer: Benjamin Staffin <benley@gmail.com>
+pkgname=jsonnet
+pkgver=0.9.3
+pkgrel=0
+pkgdesc="The data templating language"
+url="https://jsonnet.org"
+arch="all"
+license="ASL 2.0"
+depends=""
+makedepends=""
+install=""
+subpackages=""
+source="jsonnet-$pkgver.tar.gz::https://github.com/google/jsonnet/archive/v$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+  cd "$builddir"
+  CXXFLAGS="$CXXFLAGS -g -O3 -Wall -Wextra -Woverloaded-virtual -pedantic -std=c++0x -fPIC -Iinclude -Ithird_party/md5"
```

Comment:
why are you overriding CFLAGS/CXXFLAGS?  document the rationale at the very least.